k8s/resources/common/config/user-admin-server-application.yml

```
@@ -0,0 +1,3 @@
+useradmin:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)